DSharpPlus.Lavalink/LavalinkUtil.cs

```diff
@@ -98,4 +91,139 @@ private static ulong SwapEndianness(ulong v)
             return ((v & 0xFF00FF00FF00FF00) >> 8) | ((v & 0x00FF00FF00FF00FF) << 8);
         }
     }
+
+    /// <inheritdoc />
+    /// <summary>
+    /// Java's DataOutputStream always uses big-endian, while BinaryReader always uses little-endian.
+    /// This class converts a big-endian stream to little-endian, and includes some helper methods
+    /// for interacting with Lavaplayer/Lavalink.
+    /// </summary>
+    internal class JavaBinaryReader : BinaryReader
+    {
+        private static readonly Encoding Utf8NoBom = new UTF8Encoding();
```

Comment:
The name of the field here is counter to what actually does. By default `UTF8Encoding` emits a BOM. If you want BOM-less UTF-8, you need to initialize it as `new UTF8Encoding(false)`.

Reply:
Nevermind I had my docs mixed up.